new-website/utils/tutorials/export_tutorials.py

```diff
@@ -0,0 +1,182 @@
+from bs4 import BeautifulSoup
+import json
+import re
+
+
+def to_valid_identifier(s):
+    """
+    Converts a given string into a valid identifier.
+
+    Parameters
+    ----------
+    s: str
+        The string to be converted into a valid identifier.
+
+    Returns
+    -------
+    valid_identifier: str
+        The converted valid identifier.
+    """
+    valid_identifier = s.replace(" ", "_")
+    valid_identifier = re.sub(r'[^0-9a-zA-Z_]', '_', valid_identifier)
+    if valid_identifier[0].isdigit():
+        valid_identifier = "_" + valid_identifier
+    return valid_identifier
+
+
+def to_valid_url_path(s):
+    """
+    Converts a given string into a valid URL path.
+
+    Parameters
+    ----------
+    s: str
+        The string to be converted into a valid URL path.
+
+    Returns
+    -------
+    valid_url: str
+        The converted valid URL path.
+    """
+    valid_url = re.sub(r"[^\w\s]", "-", s)
+    valid_url = re.sub(r'[ _]+', '-', valid_url)
+    valid_url = valid_url.lower()
+    return valid_url
+
+
+def get_component(file_name, component_name):
+    """
+    Generates a React component for a given file name and component name.
+
+    Parameters
+    ----------
+    file_name: str
+        The name of the file to generate the component for.
+    component_name: str
+        The name of the component to be generated.
+
+    Returns
+    -------
+    generated_component: str
+        The generated React component.
+    """
+    # Remove file extension if any
+    file_name = file_name.rsplit(".")[0]
+
+    generated_component = f"""
+import TutorialLayout from "../../layouts/tutorial";
+import notebookStyles from "../../data/tutorials/styles";
+import innerHTML from "../../data/tutorials/{file_name}.js";
+import {{useEffect}} from "react";
+import scrollnav from "scrollnav";
+
+const {component_name} = () => {{
+
+useEffect(() => {{
+        document.getElementsByClassName('scroll-nav')[0]?.remove();
+        const content = document.querySelector(".notebook");
+        const insertTarget = document.querySelector(".notebook");
+
+        if (insertTarget && content) {{
+            scrollnav.init(content, {{
+                sections: "h1, h2", insertTarget: insertTarget, insertLocation: "after",
+            }});
+        }}
+
+        MathJax?.Hub?.Queue(["Typeset", MathJax.Hub]);
+    }}, []);
+
+return <div
+    className="overflow-x-scroll"
+    dangerouslySetInnerHTML={{{{__html: `${{innerHTML.html}} ${{notebookStyles}}`,}}}}
+></div>
+}}
+
+{component_name}.Layout = TutorialLayout;
+
+export default {component_name};
+"""
+
+    return generated_component
+
+
+def parse_and_export_tutorials():
+    """
+    Parses HTML files containing DeepChem tutorials and converts them into React components and JSON data that is used to
+    display tutorials on the website
+    """
+    tutorials = []
+
+    # Open 'notebooks.txt' file and iterate over each file name
+    with open('./notebooks.txt', 'r') as notebook_files:
```

Comment:
Can you add to the docstring to make a note that ./notebooks must be in this folder?